Última versão do repositório original

danilomendonca opened the pull request:
Faltavam alguns arquivos

gabrielsr committed:
Update README.md

gabrielsr committed:
Update README.md

gabrielsr committed:
Update README.md

gabrielsr committed:
Update README.md

gabrielsr committed:
Update README.md

gabrielsr committed:
Update README.md

gabrielsr committed:
Setting Java souce and compile version for 1.8 in pom.xml

gabrielsr committed:
Update README.md

gabrielsr committed:
Update README.md

gabrielsr committed:
Fixing taom4e-platform reference.

gabrielsr committed:
Merge branch 'issue_2' of github.com:lesunb/RGMToPRISM

gabrielsr committed:
Update README.md

Commit:
atualizando os pontos de extensao do plugin

Commit:
Merge branch 'master' of github.com:danilomendonca/RGMToPRISM

Conflicts:
	META-INF/MANIFEST.MF
	plugin.xml

Commit:
removendo dependência com antlr4, deixando apenas as do maven e do eclipse

gabrielsr committed:
Merge branch 'master' of github.com:lesunb/RGMToPRISM

Commit:
adicionando taom4e platform lib como dependencia

Commit:
Merge branch 'master' of github.com:lesunb/RGMToPRISM

Genaina committed:
add pluginManagement to pom.xml

Genaina committed:
Merge branch 'master' of https://github.com/lesunb/RGMToPRISM

Genaina committed:
removendo annotation de overwrite

Commit:
removendo dependencia taom4e-platform para evitar erros no classloading

Commit:
Merge branch 'master' of github.com:lesunb/RGMToPRISM

gabrielsr committed:
Update README.md

gabrielsr committed:
Update README.md

gabrielsr committed:
Update README.md

Commit:
mudança no nome do plugin e da classe de ativação

Commit:
correçao no pom.xml e import temporario do antlr na lib (o correto seria apenas usar a dependencia do mvn)

Commit:
add generated ANTLR classes

Commit:
adiciona passo para configurar o diretório de templates (futuramente será automático)

Commit:
adiciona passo para recompilar classes geradas pelo antlr ao modificar um arquivo de gramática

Commit:
refactoring do CRGMtoDTMC producer

Commit:
Refactoring e correções no gerador

Commit:
Versão com transformações sequenciais e paralelas testadas. Falta testar
demais regras.

Commit:
correções na geração com testes para tarefas sequenciais e paralelas

Commit:
generates tasks names considering their branch in the tree to avoid
repetition

Commit:
identificador das tarefas leva em consideração o id do último objetivo, permitindo repetição de ids dessas tarefas

Commit:
atualização que corrige parcialmente a geração de fórmulas de sucesso e adiciona condições de contexto de ambos os tipos para objetivos e tarefas

Commit:
add param evaluation action

Commit:
taom4eplatform update

Commit:
update preferences page labels and the readme

Commit:
README updates and no-suffix call of prism and param tools in the *PRISM/PARAM tools folder*

Commit:
Preferences page bug

Commit:
Call PRISM/PARAM from the tools folder and add scripts used by evaluation actions

* add necessary linux scripts for evaluating PRISM and PARAM
* uses the tools folder

gabrielasolano committed:
Update README.md

Adicionada seção para listar sintaxe utilizada em goals e tasks.

danilomendonca committed:
Update README.md

Correcting a few lines in the README

Commit:
Unfinished evaluation action

* used to generate parametric formulas from a CRGM and evaluate them

Commit:
Merge branch 'master' of github.com:lesunb/RGMToPRISM

danilomendonca committed:
Merge pull request #10 from gabrielasolano/patch-1

Update README.md

danilomendonca committed:
Defining the naming syntax for Goals and Tasks

danilomendonca committed:
Correcting the Goals and Tasks labeling definition

danilomendonca committed:
More labeling corrections

danilomendonca committed:
Update README.md

Commit:
add CRGM example

Commit:
Merge branch 'master' of github.com:lesunb/RGMToPRISM

danilomendonca committed:
Update README.md

danilomendonca committed:
Update README.md

Commit:
Update Eclipse version to Mars (4.5)

Commit:
Merge branch 'master' of github.com:lesunb/RGMToPRISM

Commit:
Update of the TAOM4E platform plugin .jar

Commit:
updated CRGM example

Commit:
Setting compiler-plugin version to 1.8

Commit:
Setting compiler-plugin version to 1.8

Commit:
Fix PRISM/PARAM generation

danilomendonca committed:
Merge pull request #13 from gabrielsr/master

Setting compiler-plugin version to 1.8

gabrielasolano committed:
Type checker for context annotations inserted (still with some errors)

gabrielasolano committed:
Error handling of context annotation parser.

gabrielasolano committed:
Error handling context annotations corrected

Message displayed at taom4e informing cause of error.

gabrielasolano committed:
Added antlr-complete.jar at /lib folder.

gabrielasolano committed:
CtxRegex.g4: context annotation operand allows ending with non-digit

Previously grammar only allowed operands end with digit characters.
Now it's possible to end with or without them.

gabrielasolano committed:
Code review

gabrielasolano committed:
Changed antlr-complente.jar directory

gabrielasolano committed:
Added error listener for ctx parser

Genaina committed:
adding IST paper

Genaina committed:
adding ThrowingErrorListener.java

Genaina committed:
support for typed CtxRegex

Genaina committed:
support for typed CtxRegex

Genaina committed:
updating target dir with new typed context

Genaina committed:
fixing previous commit

Genaina committed:
adding support to typed context

Genaina committed:
updating ctx model

Commit:
Pibic

yanvictords commented:
Implementação da transformação de CRGM para saída baseada em processos de negócio. 
Entrada: Modelo CRGM.
Saída: Descrição BPMN em console. 
Novas classes adicionadas e poucuíssimas alterações em código existente, sem conflitos.

Genaina committed:
Merge pull request #16 from yanvictords/master

CRGM to BPMN

Genaina committed:
new branch

Commit:
Pibic

Genaina committed:
Merge remote-tracking branch 'yan/master' into GODA_BPM

Genaina committed:
Merge branch 'GODA_BPM'

Converts CRMG into process annot..

gabrielasolano committed:
Initial code to generate parametric formula. Not tested yet.

gabrielasolano committed:
Parametric formula generation

Formulas of optional and alternative runtimes still not generated.
No contexts were taken into consideration yet.
Code optimization was made.

gabrielasolano committed:
Parametric formula generation:

Result.out has now formulas of the leaf-tasks.
Added formulas to 'opt' and 'alternative' runtime annotations.
Changes in the result.out to support the 'opt' annotation.

gabrielasolano committed:
Parametric formula generation:

Correction of error on rt_parser
Parametric formula for 'try' adapted for skip cases.

gabrielasolano committed:
Merge pull request #2 from gabrielasolano/param_formula

Param formula

Genaina committed:
Update README.md

Genaina committed:
Update README.md

Genaina committed:
Adding refs to Docker and to piStar extension.

Genaina committed:
Update README.md

Genaina committed:
Update README.md

Genaina committed:
Update README.md

Genaina committed:
Update README.md

Genaina committed:
Update README.md

Genaina committed:
Merge pull request #20 from gabrielasolano/master

Parametric Formula Generation

gabrielasolano committed:
OR-Decomposition of tasks is now correct.

PRISM modules include the or-decomposed tasks.

gabrielasolano committed:
Merge pull request #24 from gabrielasolano/master

OR-Decomposition of tasks is now correct.

gabrielasolano committed:
Adding functional tests

Adding the functional tests created in 2016 in the branch with_functional_test (not merged to avoid conflicts). A black box testing which verifies the formation of goals, tasks, context annotations and runtime annotations.

gabrielasolano committed:
Update .gitignore and remove unnecessary files

Updating .gitignore to avoid .DS_Store (from mac os) and .settings (from eclipse).
Removing these files added on previous commits.

gabrielasolano committed:
Adding functional tests

gabrielasolano committed:
Update test case to avoid false positive

gabrielasolano committed:
Update test case to avoid false positive

gabrielasolano committed:
Refactoring functional tests

Added a new class InformationRegister for better coding practice. Updated the functional tests according to this change.

gabrielasolano committed:
Runtime annotation grammar updated (N#k)

The runtime rule N#k was implemented as N%k. Now the grammar is updated and the use of this rule is according to GODA's article. The rule N#k is working properly and N%k no longer works.

RTParser and RTRegex now force antlr to throw a grammar error instead of handling it.

gabrielasolano committed:
Merge pull request #26 from gabrielasolano/master

Functional Tests and Grammar Update

Genaina committed:
adding further docs

Genaina committed:
Update README.md

Genaina committed:
adding Gabriel Nunes mono

Genaina committed:
Merge branch 'master' of https://github.com/lesunb/CRGMToPRISM

Genaina committed:
adding GabiSolanoDissertation

Genaina committed:
moving Gabi's dissertation
